yapf/__init__.py

```diff
@@ -114,7 +114,7 @@ def main(argv):
 
     try:
       reformatted_source, _ = yapf_api.FormatCode(
-          str('\n'.join(source) + '\n'),
+          str('\n'.join(source).replace("\r\n", "\n") + '\n'),
```

Comment:
Pedantic, but use single quotes.

Reply:
@Spitfire1900  sorry for my careless. aligned